back/mailer.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config();
+//step1
+const nodemailer = require('nodemailer');
+const sendEmail = function(viesti,sposti) {
+  try{    
+    // declare vars,
+    //editing the reciever to suit sending the email
+    let cutEmail = sposti.slice(11, -3);
+    let fromMail = process.env.EMAIL;
+    let toMail = cutEmail;
```

Comment:
What's the function of toMail? Just use cutEmail

Reply:
toMail is default variable name in the nodemailer. cutEmail was used to make cutting the receieved email to correct form look more simple. Deleted cutEmail and replaced it with comment explaining what is happening.